### ISSUE TITLE
Is this Gem currently being maintained?

### ISSUE THREAD

**Issue by joshbuker** [rest-client/rest-client#764]
It appears like the last update was around 2 years ago, and issues are not being resolved as well.

Does this Gem currently have any active maintainers?

**Update:** This library is unmaintained. Consider replacing with something like [Faraday](https://github.com/lostisland/faraday) ([Docs](https://lostisland.github.io/faraday/#/getting-started/quick-start))

**Comment by joshbuker** [comment]
@ab It appears you were the last one to update this project, are you still interested in maintaining it?

**Comment by n-rodriguez** [comment]
@athix I would say no.

**Comment by n-rodriguez** [comment]
https://github.com/jbox-web/rest-client

**Comment by lilith** [comment]
@ab @archiloque If this is no longer maintained, would it be possible to hand this over to new maintainers or bless a particularly maintained fork?

**Comment by h0jeZvgoxFepBQ2C** [comment]
Yeah this would be great, RestClient is very common gem and it's a pitty that you have to visit the issues page to find out that it's not actively maintained. Would be good to write this to the top of the Readme also.

**Comment by iainbeeston** [comment]
Yeah this is kinda scary, it's a [very widely used gem](https://rubygems.org/gems/rest-client/reverse_dependencies).

@n-rodriguez are you actively maintaining your fork?

**Comment by n-rodriguez** [comment]
@iainbeeston yes

**Comment by iainbeeston** [comment]
@n-rodriguez Would you be willing to make yours the official fork, or take over maintenance of this version?

It looks like @ab would need to authorize that 🤞

**Comment by n-rodriguez** [comment]
> @n-rodriguez Would you be willing to make yours the official fork, or take over maintenance of this version?

I'd like to but still waiting for @ab response...

**Comment by h0jeZvgoxFepBQ2C** [comment]
seems @ab is actively working on github repositories in general, would be great to get some feedback @ab !

**Comment by hoppergee** [comment]
Since this project has been unmaintained for a long time, and some of my projects are still relied on it. So I create a forked version and did lots of refactoring works based on it. If anyone interest, feel free to give it a try.

https://github.com/rest-man/rest-man

**Comment by drewlustro** [comment]
> **Update:** This library is unmaintained.

```
> be me
> tasked to write a ruby client for a partner API
> looks through codebase for prior art
> sees engineer used RestClient two years ago
> decide to also use RestClient, as to not introduce unnecessary complexity / divergence
> implement client
> experience issues
> go to github issues
> mfw I see this issue at the top
```

classic
